lib/pagseguro/payment_request.rb

```diff
@@ -57,6 +57,13 @@ class PaymentRequest
     # The token that identifies the request. Defaults to PagSeguro.token
     attr_accessor :token
 
+    # Determines which PagSeguro API version the request will use. Defaults to v2
+    attr_writer :api_version
+
+    # The default PagSeguro API version
+    def api_version
+      @api_version ||= 'v2'
+    end
```

Comment:
I made it thinking about being configurable. The developer decides which version he wants. 

If it not necessary, a method returning a string is actually better